enterprise/j2ee.ejbcore/src/org/netbeans/modules/j2ee/ejbcore/action/SelectMethodGenerator.java

```diff
@@ -20,10 +20,7 @@
 package org.netbeans.modules.j2ee.ejbcore.action;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
This is a problem. It breaks the jdeps(1) dependency mapping. In addition, it kills the modularity of the project.

For this reason, I have to give it a -1. Can you please fix changes like this and resubmit.